src/main/java/Cassandra.java

```diff
@@ -0,0 +1,109 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Cassandra {
+
+    private final static ArrayList<Task> taskList = new ArrayList<>();
+    private static boolean EXIT_FLAG = false;
```

Comment:
Boolean variables should be named to sound more like booleans. For instance you could say hasExit instead of EXIT_FLAG